spec/multi_email_spec.rb

```diff
@@ -53,6 +53,14 @@ class UserWithEmails < ActiveRecord::Base
         expect(user.email).to eq(nil)
       end
     end
+
+    describe 'find_by_email()' do
```

Comment:
`describe '#find_by_email`

Sorry, one small issue :-P,  we use the convention here: http://betterspecs.org/#describe
`#` for instance methods, and `.` for class methods.